client/js/components/faq/DpFaqSupport.vue

```diff
@@ -0,0 +1,118 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <div
+    id="scFaq"
+    class="c-support-wrapper space-inset-m">
+    <h2 class="font-normal">
+      {{ Translator.trans('support.heading') }}
+    </h2>
+    <p class="mb-5">
+      {{ Translator.trans('support.introduction') }}
+    </p>
+    <p>
+      {{ Translator.trans('support.contact.advice') }}
+    </p>
+    <h3 class="mt-5">
+      {{ Translator.trans('support.content') }}
+    </h3>
+    <ul class="grid lg:grid-cols-3 gap-3">
+      <li
+        v-for="contact in contacts"
+        :v-key="contact.id"
+        class="space-inset-m h-48 c-support-card">
+        <dp-faq-support-card>
+          <template v-slot:title>
+            <h4 class="font-semibold">
+              {{ contact.attributes.title }}
+            </h4>
+          </template>
```

Comment:
Why do You use slots and not props?
(here and below)

Reply:
done https://github.com/demos-europe/demosplan-core/pull/2139/commits/9de7d2e1c129d7e25c4b79cc65f559d78d5efb4d